### DISCUSSION TITLE
How can I create a Markdown link based on the current tab and copy it to the clipboard in one key binding?

### DISCUSSION THREAD

**Discussion by maxigaz** [gdh1995/vimium-c#938] [Q&A] [answered]
I would like to create a key binding in Vimium C running in Firefox that takes the title and URL of the current tab, formats it so that it looks like a Markdown link (such as `[Page title](http://example.com)`), and finally copy that to the clipboard. Is this possible to do with a single key mapping (e.g. by pressing `yY`)?

Here are all the related options I'm already aware of:

- `map yY copyWindowInfo format="[${title}](${url})"`: This copies information of *every* tab inside the current window, not just the current tab.
- `map yY copyCurrentTitle`: Only copies the title.
- `map yY copyCurrentUrl`: I see that it has the option `type=url/title/frame`, but I don't know if I can use this for copying *both* the URL and the title at the same time.
- `map yY openUrl url="javascript:..."`: I don't know if Firefox even allows this through pure Javascript. If it does, any pointers on how to implement this would be apreciated.

**Comment by gdh1995** [comment, marked as answer]
You just need ```map <f6> copyWindowInfo type="tab" format="[${title}](${url})"``` - `copyWindowInfo` use `type` to select a target tab, window, or other things like a frame.
